src/bors/handlers/review.rs

```diff
@@ -135,6 +153,23 @@ async fn notify_of_unapproval(repo: &RepositoryState, pr: &PullRequest) -> anyho
         .await
 }
 
+async fn notify_of_edited_pr(
+    repo: &RepositoryState,
+    pr_number: PullRequestNumber,
+    head_sha: CommitSha,
+) -> anyhow::Result<()> {
+    repo.client
+        .post_comment(
+            pr_number,
+            Comment::new(format!(
+                r#":warning: The base branch changed to `{}`, and the 
```

Comment:
This message doesn't make sense if the PR was not approved in the first place. We should only send this message when the PR was originally approved.